lib/index.js

```diff
@@ -138,13 +138,15 @@ module.exports = Serializer => {
 
       contextResponse.payload = {
         [reservedKeys.jsonapi]: jsonapi,
+        [reservedKeys.meta]: {},
```

Comment:
Might seem weird, but all responses **must** contain either data, errors, or meta. We definitely have no data nor errors at the root, so I think it makes the most sense to include an empty `meta`.